src/test/resources/uk/org/okapibarcode/output/code93-with-rotation-90.eps

```diff
@@ -0,0 +1,77 @@
+%!PS-Adobe-3.0 EPSF-3.0
```

Comment:
This EPS file also not correct (beyond just the clockwise / counter-clockwise issue).